### ISSUE TITLE
Enhancement: Save As... shouldn't reset Configuration per file...

### ISSUE THREAD

**Issue by nerdfever** [spyder-ide/spyder#15232]
Save As... in the Spyder editor, shouldn't reset the the Run>Configuration per file... info. That info should carry over to the newly named source file.

Because you're saving the file "as" another filename, not doing a "Save copy as...", all settings associated with the editor file should remain the same.

I use a primitive version control scheme where periodically I save the file under a new name ("program10.py, program11.py, program12.py") in case I need to go back to an earlier version because I screwed up somehow.

This misfeature (reset of the info) is unfortunate - the Configuration per file... box pops up on the first run of each renamed file,  often necessitating tedious re-entry of the command line options.

**Comment by ccordoba12** [comment]
Good idea. You're welcome to try to fix it because (unfortunately) we don't have time to do it, sorry.

**Comment by nerdfever** [comment]
I'm well aware you guys are going nuts with Spyder 5 bugs.

This is, obviously, low priority and can wait (that's why I said "Enhancement:"). But best to get the ticket in while I'm thinking of it.

Unfortunately I have neither the time not skill to help do it myself - I did just toss a few $$ into the kitty to help. :-)
